browser.js

```diff
@@ -0,0 +1,11 @@
+var EventEmitter = require('events').EventEmitter
+var getBrowserRTC = require('get-browser-rtc')
+
+module.exports = function (opts) {
+  var wrtc = new EventEmitter()
+  // If electron-webrtc is required in the browser, return the browser RTC implementation.
+  return Object.assign(wrtc, getBrowserRTC(), {
```

Comment:
Ah, nevermind then. Last time I checked it wasn't supported in Safari so I have made a habit of using `object-assign`, but it seems that has changed.